app/controllers/rentals_controller.rb

```diff
@@ -0,0 +1,64 @@
+class RentalsController < ApplicationController
+  SORT_CATEGORIES = ["movie_id", "title", "customer_id", "name", "postal_code", "checkout_date", "due_date"]
+
+  def create
+    if params[:movie_id].nil? || params[:customer_id].nil?
+      render json: { ok: false, "errors" => ["unable to create rental"]}, status: :bad_request
+      return
+    end
+    if Movie.find(params[:movie_id]) && Customer.find(params[:customer_id])    
```

Comment:
Lines 9-12 could appropriately be pulled out into 1 or 2 model methods. but that's getting on the nit-picky side for sure.